package.json

```diff
@@ -21,12 +21,15 @@
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.2",
     "eslint-plugin-react": "^7.5.1",
-    "nodemon": "^1.12.1"
+    "nodemon": "^1.12.1",
```

Comment:
hi, the changes to `package.json` and `package-lock.json` still need to be removed please.

Also, you need to finish the changes to eslintrc - it's renamed in `examples/webpages` and shouldn't be; and you will need another one in `examples/tests`